add the necessary flow to create and recover an ETH account:

ChrisYnz opened the pull request:
- The account is based in a 12-word mnemonic seed
- A password is mandatory to encrypt the seed.
- For now, the wallet is not used for anything

![addWallet](https://user-images.githubusercontent.com/53454404/117898364-0d167400-b282-11eb-8211-a7aaeda4b638.png)
![confirmPassword](https://user-images.githubusercontent.com/53454404/117898381-17d10900-b282-11eb-8722-519b88dff3d0.png)
![seedPhrase](https://user-images.githubusercontent.com/53454404/117898393-1e5f8080-b282-11eb-8628-dfabaa9ae15a.png)
![confirmSeedPhrase](https://user-images.githubusercontent.com/53454404/117898407-23bccb00-b282-11eb-8549-899b34132d38.png)
![unlockWallet](https://user-images.githubusercontent.com/53454404/117898488-4d75f200-b282-11eb-92c3-405feb53dad9.png)
![restoreWallet](https://user-images.githubusercontent.com/53454404/117898412-28817f00-b282-11eb-9887-37b75c332275.png)

AlexITC reviewed (approved):
@roguealexander approved this but forgot to click on the button.

ChrisYnz committed:
add the necessary flow to create and recover an ETH account:

- The account is based in a 12-word mnemonic seed
- A password is mandatory to encrypt the seed.
- For now, the wallet is not used for anything
